### DISCUSSION TITLE
make Azuracast the webend for mixxx

### DISCUSSION THREAD

**Discussion by devrob20** [AzuraCast/AzuraCast#5692] [Ideas]
i do like the functions of azuracast, but it currently has some quirks. it does like all its content local, which makes is painfully slow and buggy when you use SSH/sftp.  the webDJ has some use-cases, but in real life ppl tend to use dedicated hardware and software like RadioDJ and MiXXX, the first one has also a songrequest wordpress plugin, but needs windows to run, MiXXX doesn't have an songrequest, but does run on all windows/mac/linux distro's.

so focussing the development of Azuracast towards making it a api bridge for MiXXX would be a great benefit, ppl still could use the autoDJ, but also take song request frm the web via the MiXXX software, and update twitter, telegra, rocket.chat or any other platform they have via the azuracast webhooks and use the web server as webapp.

**Comment by ItsMitchh** [comment]
While this does sound mildly interesting, can you give some more information on Mixxx's API and/or this 'bridge API' you mention?

**Comment by Moonbase59** [comment]
Hm, I don’t seem to get the problem. Just fired up my old Mixxx (1.10.1 still), entered the Icecast 2 DJ connection data for my station and voilà—online and mixing, metadata, DJ image and all.

Haven’t used Mixxx for quite a while, although it’s great for real DJs, mixing a live set. For Radio-like broadcasts, I typically use IDJC (Linux; JACK req'd; no SMB/CIFS, NFS is fine).
